cmd/main.go

```diff
@@ -109,7 +114,8 @@ func main() {
 	secretManagers := map[string]secret.SecretManager{
 		"raw": secret.RawSecretManager{},
 	}
-	if cloudSecretManagerType == "gcp" {
+	switch cloudSecretManagerType {
```

Comment:
i didn't expect to have k8s secret at the beginning, so cloudSecretManagerType would be not appropriate?
`adminUserSecretType` or sth?

Reply:
Sounds good to me